.github/workflows/pr-check.yaml

```diff
@@ -223,24 +219,65 @@ jobs:
       - name: Run formatter
         run: pnpm format:check
 
-      - name: Run unit tests
-        run: pnpm test:unit
-
-      - name: Run typecheck
-        run: pnpm typecheck
-
-      - name: Run svelte check
-        run: pnpm svelte:check
-
       # Check we don't have changes in git
       - name: Check no changes in git
-        if: ${{ matrix.os=='ubuntu-24.04'}}
         run: |
           if ! git diff --exit-code; then
             echo "Found changes in git"
             exit 1
           fi
 
+  tpecheck:
```

Comment:
not like it matters a lot, but there is a typo: just missing `y`

Reply:
thanks I think I need to change my glasses !